src/Modules/ContentBlocks/Backend/Actions/ContentBlockEdit.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace ForkCMS\Modules\ContentBlocks\Backend\Actions;
+
+use ForkCMS\Core\Domain\Header\FlashMessage\FlashMessage;
+use ForkCMS\Modules\Backend\Domain\Action\AbstractFormActionController;
+use ForkCMS\Modules\Backend\Domain\Action\ActionSlug;
+use ForkCMS\Modules\ContentBlocks\Domain\ContentBlock\Command\ChangeContentBlock;
+use ForkCMS\Modules\ContentBlocks\Domain\ContentBlock\ContentBlock;
+use ForkCMS\Modules\ContentBlocks\Domain\ContentBlock\ContentBlockType;
+use Symfony\Component\HttpFoundation\RedirectResponse;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+
+/**
+ * Edit an existing ContentBlock.
+ */
+final class ContentBlockEdit extends AbstractFormActionController
+{
+    protected function getFormResponse(Request $request): ?Response
+    {
+        /** @var ContentBlock $contentBlock */
+        $contentBlock = $this->getEntityFromRequest($request, ContentBlock::class);
+
+        $this->assign('content_block', $contentBlock);
+
+        if ($this->getRepository(ContentBlock::class)->count([]) > 1) {
```

Comment:
Wh did you add this if statement here?
I think it can always be shown

Reply:
fixed in https://github.com/forkcms/forkcms/pull/3555/commits/b6be7317c9d4208d8d233d4035b7e4dd95c1a68e